### ISSUE TITLE
Close mic on response

### ISSUE THREAD

**Issue by Iridio** [dialogflow/dialogflow-fulfillment-nodejs#30]
How can I tell Dialogflow to not expect a response from the user (close the mic)? 
In your samples I saw only the use of agent.add() with phrases that expect always a response.
Something like ask/tell (now ask/close) in actions-on-google.

**Comment by segaura** [comment]
+1
In DialogFlow console we have the switch to enable "Set this intent as end of conversation" after a text response.

My guess is that the answer relates to the DialogflowConversation object, agent.conv() in the following excerpt

```
const { WebhookClient } = require('dialogflow-webhook');
const agent = new WebhookClient({request: request, response: response});
let conv = agent.conv();
conv.ask('Hi from the Actions on Google client library');
agent.add(conv);
```

is there any DialogflowConversation documentation?

**Comment by Iridio** [comment]
Exactly, I know there is the switch on the backend, but I need to handle it at runtime from my webhook.
From actions-on-google is easy because of `ask()` and `close()`

**Comment by jorgecasar** [comment]
@Iridio the concept of "End Conversation" is Actions on Google feature. As you can read in [Actions on Google: Build Fulfillment](https://developers.google.com/actions/dialogflow/fulfillment)

> **Actions on Google** lets you specify platform-specific settings for the intent. Currently, only **End conversation** is supported. This closes the mic and ends the conversation after the corresponding response is returned to the user. This setting only honored if you don't use fulfillment for the intent.

Then, if you want to implement in your own client you have to create a custom payload or read google's one.
